db/migrate/20210203152832_add_process_id_to_practice_editor_sessions.rb

```diff
@@ -0,0 +1,5 @@
+class AddProcessIdToPracticeEditorSessions < ActiveRecord::Migration[5.2]
+  def change
+    add_column :practice_editor_sessions, :process_id, :bigint, null: true
+  end
+end
```

Comment:
Instead of creating another migration, we can just do a rollback and add this column to the migration above

Reply:
I have a migration in between these two from merging master branch in here.  a new migration to add/remove columns is not an issue.